src/webview/create-component/createComponentLoader.ts

```diff
@@ -232,8 +235,8 @@ export default class CreateComponentLoader {
                             // move the cloned git repo to selected project path
                             componentFolder = path.join(message.data.gitDestinationPath, componentName);
                             await fs.mkdir(componentFolder);
-                            await fse.copy(message.data.tmpDirUri.fsPath, componentFolder);
-                            await fs.rm(message.data.tmpDirUri.fsPath, { force: true, recursive: true });
+                            await fse.copy(tmpFolder.fsPath, componentFolder);
+                            await fs.rm(tmpFolder.fsPath, { force: true, recursive: true });
```

Comment:
I get the following in the extension host logs if I step through all the "import from git" steps:

```
rejected promise not handled within 1 second: Error: ENOENT: no such file or directory, stat '/tmp/tmp-23687-2JNsu65ekALv'
```

I think delete is being triggered twice on the folder, once when you move the contents and another time when the folder is emptied. The second time fails with an error message in the console, since the directory doesn't exist.

I think it would be nice to avoid this, although it's not a huge deal, since it's hidden from the user (no pop ups)